### PR TITLE
Add 1.3 release notes

### PR DESCRIPTION
**Pull request by eugenesk24**
Signed-off-by: Eugene Lee <eugenesk@amazon.com>

### Description
Add 1.3 release notes
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+### Version 1.2.4.0 Release Notes
```
Comment (penghuo) [inline release-notes/opensearch-sql.release-notes-1.3.0.0.md:1]: Version 1.3.0 Release Notes?

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Documentation
+* Add parse docs to PPL commands index ([#486](https://github.com/opensearch-project/sql/pull/486))
+* Add limitation section in PPL docs ([#456](https://github.com/opensearch-project/sql/pull/456))
+* Add how to setup aws credentials for ODBC Tableau ([#394](https://github.com/opensearch-project/sql/pull/394))
+
+### Maintenance
+* Add JDK 8 to CI Matrix  ([#483](https://github.com/opensearch-project/sql/pull/483))
+* Add CI Matrix for JDK 11 and 14 ([#451](https://github.com/opensearch-project/sql/pull/451))
+* Update backport and add auto-delete workflows ([#446](https://github.com/opensearch-project/sql/pull/446))
+* Add auto backport functionality for SQL ([#445](https://github.com/opensearch-project/sql/pull/445))
```
Comment (joshuali925) [inline release-notes/opensearch-sql.release-notes-1.3.0.0.md:31]: similar comment as in the observability one: CI/workflows/tests are in infrastructure, third-party library version bumps are usually to fix CVEs so they are bug fixes. Also 479 is a bug fix, others look good to me

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+### Version 1.2.4.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0
+
+### Enhancements
+* PPL integration with AD and ml-commons ([#468](https://github.com/opensearch-project/sql/pull/468))
```
Comment (joshuali925) [inline release-notes/opensearch-sql.release-notes-1.3.0.0.md:6]: also not sure if this is going in? this and 411 can be features

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md (hunk 1 above)
Comment (eugenesk24) [reply release-notes/opensearch-sql.release-notes-1.3.0.0.md:1]: 😆 Yes yes mistake

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md (hunk 3 above)
Comment (eugenesk24) [reply release-notes/opensearch-sql.release-notes-1.3.0.0.md:6]: @penghuo Is this not going in?

**Comment by eugenesk24** [comment]
Closing PR and opening identical [PR](https://github.com/opensearch-project/sql/pull/490) pushed from fork

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md (hunk 3 above)
Comment (penghuo) [reply release-notes/opensearch-sql.release-notes-1.3.0.0.md:6]: yes, parse command release in 1.3

Comment (joshuali925) [reply release-notes/opensearch-sql.release-notes-1.3.0.0.md:6]: #468, not parse, is AD ml-commons going in?

**Pull request closed without merging**